### ISSUE TITLE
Position two-way databinding

### ISSUE THREAD

**Issue by gentle9** [swiety85/angular2gridster#116]
I am trying to save the state of gridster to a file. For that I need the positions and size.
The two-way databinding is working for the size but for the position the value is not changed after the position was changed.
I have a changeEvent which is triggered correctly. In this function I am printing out the widgets array to see what has changed.
When I change the size of a widget the value of w and h is changed correctly in the array. But when I change the position of the widget the value of x and y is not changed at all.

This is my Widgets array.
```
widgets:any = [
    {
        x: 0, y: 0, w: 1, h: 2,
        title: 'Basic form inputs 1',
        content: 'Widget content'
    }
]
```

**Comment by swiety85** [comment]
Hi, do you have responsiveOptions defined in your gridster configuration?

**Comment by swiety85** [comment]
If you set responsive options and you want to have two way binding, then you should provide attributes for each breakpoint defined in `responsiveOptions`.
Default `x`, `y` with `responsiveView` are responsible for the smallest breakpoint (by default till 576px window width). I suppose you have bigger window width. Therefore Gridster will try to assign position to attributes dedicated for current breakpoint. [Here you have an example.](https://github.com/swiety85/angular2gridster/wiki/API-Documentation#html-example-1)

**Comment by gentle9** [comment]
Ah ok I didn´t see that. I have changed my Widget array. But this seems to work only for a static array.

If i have the following code the value of x and y is preferred even if the window width is XL. This means that on moving a widget only the value of xXl is changed. But when loading this widget again the value of x is taken instead of xXl.

This works. Widget position on screen is 5.
```
widgets:any = [
    {
        x: 0, y: 0, w: 1, h: 2, xSm: 0, ySm: 0, xMd: 0, yMd: 0, xLg: 0, yLg: 0, xXl: 5, yXl: 0,
        title: 'Basic form inputs 1',
        content: 'Widget content'
    }
]
```

This doesn´t work. The position is loaded from x instead from xXl. Widget position on screen is 0 (should be 5).
```
widgets:any = [];
widgetsTemp:any = [
    {
        x: 0, y: 0, w: 1, h: 2, xSm: 0, ySm: 0, xMd: 0, yMd: 0, xLg: 0, yLg: 0, xXl: 5, yXl: 0,
        title: 'Basic form inputs 1',
        content: 'Widget content'
    }
]
...
this.widgets = this.widgetsTemp;
...
```

**Comment by swiety85** [comment]
Please check if problem still occurs in v 0.6.9.

**Comment by gentle9** [comment]
I have tested it with the current version v 1.0.0-alfa and the problem still occurs. It still takes only the x and y values instead of the responsive value when the widgets are added after initialization.

**Comment by swiety85** [comment]
I can't reproduce your problem can you try to create plunker example, or at lease pass me your controller including gridster configruration and the function that you assign widgets array. Is assigning widgets happens in async function?

**Comment by swiety85** [comment]
Problem should be fixed in v1.0.1
